Lil-Peep/src/styles/modules/pdp.scss

```diff
@@ -0,0 +1,243 @@
+    align-items: center;
+    display: flex;
+    height: 100%;
+    width: 50%;
+  }
+}
+
+.super-product {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  .box-product {
+    display: flex;
+    height: 300px;
+    justify-content: center;
+    width: 100%;
+
+    .product {
+      height: 100%;
+      justify-content: center;
+      object-fit: contain;
+      width: 100%;
+    }
+
+    @media #{$breakpoint-tablet} {
+      width: 100%;
+    }
+  }
+
+  .box-micro {
+    display: flex;
+    justify-content: center;
+    padding-top: 25px;
+    width: 70%;
+
+    a {
+      height: 85px;
+      margin-right: 20px;
+      width: 85px;
+
+      img {
+        border-color: transparent;
+        border-style: solid;
+        border-width: 5px;
+        object-fit: contain;
+      }
+
+      &:last-child {
+        margin-right: 0;
+        margin-top: 0;
+      }
+
+      @media #{$breakpoint-tablet} {
+        height: auto;
+        max-width: 100%;
+      }
+    }
+
+    .selected {
+      border-color: #{$view-green};
+      border-radius: 5px;
+      border-style: solid;
+      border-width: 5px;
+    }
+
+    .view {
+      height: 85px;
+      object-fit: cover;
+      width: 85px;
+
+      @media #{$breakpoint-tablet} {
+        height: 70px;
+        object-fit: cover;
+        width: 70px;
+      }
+    }
+
+    @media #{$breakpoint-tablet} {
+      align-self: flex-start;
+      flex-direction: column;
+      height: 100%;
+      justify-content: flex-start;
+      padding-top: 0;
+      width: 150px;
+    }
+  }
+
+  @media #{$breakpoint-tablet} {
+    flex-direction: row;
+    height: 100%;
+    justify-content: space-between;
+    width: 100%;
+  }
+}
+
+  .half-right {
+    display: flex;
+    flex-direction: column;
+
+    .product-title {
+      align-self: center;
+      color: #{$title-font-gray};
+      font-family: 'Lato', sans-serif;
+      font-weight: 100;
+
+      @media #{$breakpoint-tablet} {
+        align-self: flex-start;
+        color: #{$title-font-black};
+      }
+    }
+
+    .info-container {
+      align-items: center;
+      display: flex;
+      flex-direction: column;
+
+      .product-quantity {
+        flex-direction: row;
+        margin-top: 20px;
+
+        label {
+          color: #{$title-font-gray};
+          font-family: 'Lato', sans-serif;
+        }
+      }
+
+      @media #{$breakpoint-tablet} {
+        align-items: flex-start;
+      }
+    }
+
+    .product-price {
+      color: #{$price-purple};
+      font-family: 'Lato', sans-serif;
+      font-size: 25px;
+      font-weight: bold;
+      font-weight: 100;
+      text-align: center;
+
+      .old-price {
+        color: #{$price-purple};
+      }
+    }
+
+    .add-to-cart {
+      background-color: #{$view-green};
+      border-radius: 3px;
+      border-style: none;
+      margin-left: 50px;
+      margin-right: 50px;
+      margin-top: 20px;
+      padding: 10px 0 10px 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `10px 0` instead of `10px 0 10px 0`